docs/adguard-for-android/solving-problems/logcat.md

```diff
@@ -11,6 +11,8 @@ This article is about AdGuard for Android, a multifunctional ad blocker that pro
 
 Sometimes a regular log may not be sufficient to identify the origin of the problem. In such cases a system log is needed. Below are instructions on how to collect and get it: via Developer options and Logcat.
 
+> AdGuard is committed to protecting your privacy. We strictly follow our [Privacy Policy](https://adguard.com/en/privacy/android.html) and do not collect any private information about users. Before sending your logs to the support team, please review the file as it may contain additional information that you don’t want to share. If it contains such personal information, we recommend that you delete it first.
```

Comment:
let's use an admonition (:::info/:::note) here instead of the blockquote (>) and remove the English locale from the link
please change this text in other sections (AdGuard Browser Extension, AdGuard for Android, and AdGuard for Windows) as well